src/SmartComponents/Packages/Packages.test.js

```diff
@@ -64,7 +64,12 @@ afterEach(() => {
 
 describe('Packages.js', () => {
     it('should match the snapshot', () => {
-        expect(toJson(wrapper)).toMatchSnapshot();
+        const { asFragment } = render(
```

Comment:
This test is useless as well. It doesn't even render any packages[0].
Let's drop this one, and maybe extend the test below and verify a list of pacakges is actually shown after it dispatched fetching.

[0] https://github.com/RedHatInsights/patchman-ui/pull/1123/files#diff-c738ba934bfc4b8dba39e7de8858f7aeaa0f81987ec0e3e5a62a1440289c9b80R261

Reply:
Added 2 tests to check the packages names and summary